vulnerabilities/package_managers.py

```diff
@@ -347,22 +429,51 @@ class GitHubTagsAPI(VersionAPI):
     package_type = "github"
 
     async def load_api(self, repo_set):
-        async with client_session() as session:
+        session = client_session()
+        async with session as session:
```

Comment:
This code looks hard to read... we should have intermediate variable to avoid the nested complexity, possibly using generator expressions.

Reply:
Tracking this in https://github.com/nexB/vulnerablecode/issues/492 . 

Could you elaborate an approach there ?